hydradx/model/indexer_utils.py

```diff
@@ -40,12 +46,14 @@ def query_indexer(url: str, query: str, variables: dict = None) -> dict:
     return return_val
 
 
-def get_asset_info_by_ids(asset_ids: list) -> dict:
+def get_asset_info_by_ids(asset_ids: list = None) -> dict:
 
-    asset_query = """
-    query assetInfoByAssetIds($assetIds: [String!]!) {
-      assets(filter: {id: {in: $assetIds}}) {
-        nodes {
+    asset_query = f"""
+    query assetInfoByAssetIds{'($assetIds: [String!]!)' if asset_ids else ''} {{
+      assets(filter: {{{
+        'id: {in: $assetIds}, ' if asset_ids else ''
+      }symbol: {{notEqualTo: "none"}}}}) {{
```

Comment:
what's with the doubling up on curly braces?

Reply:
It's for the formatting string. Double curly braces = literal curly brace, single = insert value of expression inside. I did this because in instances where you want all assets (not only certain ones) you can skip doing a query to get all asset ids and simplify this query by not including that filter.

Reply:
Interesting, I'm not sure I quite follow but I'll take a look. Sounds good, I was doing a whole separate query for that case (not filtering on asset ID).